src/modules/siptrace/siptrace.c

```diff
@@ -1607,6 +1608,11 @@ static void trace_onreply_out(struct cell *t, int type, struct tmcb_params *ps)
 	if(msg == NULL || msg == FAKED_REPLY) {
 		msg = t->uas.request;
 		faked = 1;
+		/* check if from header has been already parsed.
+		 * If not we have to parse it in pkg memory and free iit at the end.
+		 */
+		if (msg->from && msg->from->parsed != NULL)
```

Comment:
Should the above condition have:

```
		msg->from->parsed == NULL
```

Otherwise it should be already parsed.

Reply:
you're right, I wrongly copied from local. I'm going to fix and force push.